package.json

```diff
@@ -29,6 +29,7 @@
   "main": "scripts/index.js",
   "dependencies": {
     "angular-history": "git+https://github.com/decipherinc/angular-history.git#v0.8.0",
+    "superdesk-planning": "git+https://github.com/superdesk/superdesk-planning.git#1f04215",
```

Comment:
I added it here because last time I tried (before your #995) it was impossible to add it at the `superdesk/superdesk` level. Ideally, in my opinion, it would be loaded there in order to be included in integration tests. But the feature must stay option-able in term of a user's point of view.